invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -253,10 +253,8 @@ export const RequestToggleComponent = ({
       return;
     }
     setOpen(true);
-    const filterToRemove = userSelectionFilters.filter((obj1) => {
-      return obj1.includes("is_open");
-    });
-    const filters = filterToRemove ? filterToRemove : [];
+    // We add the selected filters by the user to remove them
+    const filters = [...userSelectionFilters];
```

Comment:
Is it ok to remove all the filters or should we only remove the `Status` filters?
To me it makes sense to remove all of them

Reply:
To keep them all is not an option at the moment since it's the reason of the bug. The issue happens when we click a status such as "open" and then we click "Closed" filter. See below an example.
![filters_bug](https://user-images.githubusercontent.com/15194802/153150580-58f226c6-df55-4d3a-b79a-98756e382a90.gif)

Based on this we can or remove all filters as it's happening in this PR or just remove the satus filters and keep the type filters.